nix-in-docker/run

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+repo_dir="$(cd "$(dirname "$0")/.." && pwd)"
+
+docker run \
+  -it \
+  --rm \
+  -v=$repo_dir:/mantis-ops \
+  -w=/mantis-ops \
+  -v=mantis-ops-root:/root \
+  -v=mantis-ops-nix:/nix \
```

Comment:
Using a separate directory here means that the users `/nix` can't be shared with the one in Docker. Not sure if there's a way around that though.

Reply:
I tried sharing it with my own `/nix` mount but that produced some permission errors, so this is probably easiest. (This script is also purely for convenience.)